src/test/java/com/google/api/generator/gapic/dummy/FileDiffInfraDummyTest.java

```diff
@@ -0,0 +1,104 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.api.generator.gapic.dummy;
+
+import static junit.framework.Assert.assertEquals;
+
+import com.google.api.generator.engine.ast.BlockComment;
+import com.google.api.generator.engine.ast.ClassDefinition;
+import com.google.api.generator.engine.ast.CommentStatement;
+import com.google.api.generator.engine.ast.LineComment;
+import com.google.api.generator.engine.ast.ScopeNode;
+import com.google.api.generator.engine.writer.JavaWriterVisitor;
+import java.util.Arrays;
+import org.junit.Test;
+
+public class FileDiffInfraDummyTest {
+  // Add two simple tests for testing the file-diff infra.
+  // 1. The two unit tests create simple java classes.
+  // 2. Two unit tests are created because we have the case of two expected strings comparison
+  // in one test class e.g. ResourceNameHelperClassComposer. In that case, two golden files will be
+  // created.
+  //
+  // TODO(xiaozhenliu): remove this test class once the file-diff infra is in place and well-tested.
+  @Test
+  public void simpleClass() {
+    ClassDefinition classDef =
```

Comment:
I removed the composer layer in the test class, only testing classDef node. Because it's not in gapic/composer folder any more, and the GapicClass/addApacheLicense is package visible. So Apache License is manually added here to avoid reusing the helper from composer folder.

Reply:
SGTM.